include/functions.h

```diff
@@ -570,19 +570,19 @@ void CollisionPoly_GetVerticesByBgId(CollisionPoly* poly, s32 bgId, CollisionCon
 // ? BgCheck_SphVsStaticWall(?);
 s32 BgCheck_CheckStaticCeiling(StaticLookup* lookup, u16 xpFlags, CollisionContext* colCtx, f32* outY, Vec3f* pos,
                                f32 checkHeight, CollisionPoly** outPoly);
-s32 func_8003A5B8(SSList* headNodeId, CollisionContext* colCtx, u16 xpFlags1, u16 xpFlags2, Vec3f* posA, Vec3f* posB,
+s32 BgCheck_CheckLineAgainstSSList(SSList* headNodeId, CollisionContext* colCtx, u16 xpFlags1, u16 xpFlags2, Vec3f* posA, Vec3f* posB,
                   Vec3f* outPos, CollisionPoly** outPoly, f32* outDistSq, f32 chkDist, s32 bccFlags);
-// ? func_8003A7D8(?);
+// ? BgCheck_CheckLineInSubdivision(?);
 // ? BgCheck_SphVsFirstStaticPolyList(?);
 // ? BgCheck_SphVsFirstStaticPoly(?);
 // ? BgCheck_GetNearestStaticLookup(?);
 // ? BgCheck_GetStaticLookup(?);
 void BgCheck_GetStaticLookupIndicesFromPos(CollisionContext* colCtx, Vec3f* pos, Vec3i* arg2);
-// ? func_8003AEA8(?);
-// ? func_8003B04C(?);
-// ? func_8003B218(?);
-// ? func_8003B3C8(?);
-// ? func_8003BB18(?);
+// ? BgCheck_GetSubdivisionMinBounds(?);
+// ? BgCheck_GetSubdivisionMaxBounds(?);
+// ? BgCheck_GetPolySubdivisionBounds(?);
+// ? BgCheck_PolyIntersectsSubdivision(?);
+// ? BgCheck_InitializeStaticLookup(?);
```

Comment:
for all of these commented prototypes, the should either be added here or deleted if they are internal functions to bgcheck not meant to be called externally